mirrord/cli/src/execution.rs

```diff
@@ -195,7 +198,7 @@ impl MirrordExecution {
 
         let stdout = proxy_process.stdout.take().expect("stdout was piped");
 
-        let port: u16 = BufReader::new(stdout)
+        let socket: SocketAddr = BufReader::new(stdout)
```

Comment:
```suggestion
        let address: SocketAddr = BufReader::new(stdout)
```

I would prefer this, as `socket` carries more a ... well ... socket meaning.